src/pages/zh-cn/getting-started.mdx

```diff
@@ -66,7 +77,7 @@ pnpm create astro@latest
 
 📚 探索 Astro 的[运行时 API](/zh-cn/reference/api-reference/)。
```

Comment:
Looks like this bullet point and the two above also need updating. Could you update these as part of this PR?

Here’s the current English version:

```md
See examples of some of the key concepts and patterns of an Astro site!

📚 [Add your first page](/en/core-concepts/astro-pages/) to your site.

📚 Read more about Astro’s [project structure](/en/core-concepts/project-structure/).

📚 Learn about Astro's [file-based routing](/en/core-concepts/routing/).

*... find our full API documentation under the **Reference** tab.*
```

Reply:
Oh, I missed it. It has been updated now